src/nvim/CMakeLists.txt

```diff
@@ -791,6 +792,22 @@ add_custom_target(
   DEPENDS ${LINT_PRG} ${LINT_NVIM_SOURCES} ${LINT_SUPPRESS_FILE}
 )
 
+add_custom_target(
+  clint-non-style
+  COMMAND
+    ${LINT_PRG} --filter=-whitespace,-readability ${LINT_NVIM_REL_SOURCES} 
```

Comment:
this is not worth adding as a build target, it is just `clint.py` with some arbitrary options.

Reply:
This does not use the `--supress-errors` flag. I thought it could end beng more performant, but I didn't really benchmark it. Anyways, this is totally disposable. I just wanted to make it clear how to get a clearer baseline.